client/rest-high-level/src/main/java/org/elasticsearch/client/ml/inference/preprocessing/FrequencyEncoding.java

```diff
@@ -154,7 +166,7 @@ public Builder addFrequency(String valueName, double frequency) {
         }
 
         public FrequencyEncoding build() {
-            return new FrequencyEncoding(field, featureName, frequencyMap);
+            return new FrequencyEncoding(field, featureName, frequencyMap, null);
```

Comment:
Whey not add `setCustom(Boolean isCustom)` to the builder?

Reply:
@davidkyle I wanted to wait on this until we have support for custom processors in analytics. Then we can see how it all plays together. 

Adding a builder setter is not a big deal. I can add one now.

Reply:
> Then we can see how it all plays together.

That's good.